build/android/config.gni

```diff
@@ -44,6 +44,8 @@ brave_chrome_java_deps = [
   "//brave/third_party/android_deps:com_android_installreferrer_java",
   "//brave/third_party/android_deps:com_brave_playlist_java",
   "//brave/third_party/android_deps:com_github_bumptech_glide_java",
+  "//brave/third_party/android_deps:com_google_android_datatransport_transport_backend_cct_java",
```

Comment:
we should not be adding indirect deps to this list. Only add direct deps here (the java imports you actually reference). Indirect deps are handled automatically and should not be added individually.